File: ExpenseManager/EM.API/Models/Category.cs
using System;

namespace EM.API.Models
{
    public class Category
    {
        public Guid CategoryID { get; set; }
        public Guid TransactionTypeID { get; set; }
        public string TransactionTypeName { get; set; }
        public string CategoryName { get; set; }
        public int Sequence { get; set; }
        public DateTime CreatedDate { get; set; }
    }
}

File: ExpenseManager/EMWEB/src/app/models/category.ts
export class Category {
    CategoryID: string;
    TransactionTypeID: string;
    TransactionTypeName: string;
    CategoryName: string;
    Sequence: number;
    CreatedDate: Date;

    constructor(object?:any) {
        if(object) {
            Object.assign(this, object)
        }
    }
}  